libs/application/templates/official-journal-of-iceland/src/fields/Submitted.tsx

```diff
@@ -0,0 +1,24 @@
+import { FormGroup } from '../components/form/FormGroup'
+import { OJOIFieldBaseProps } from '../lib/types'
+import { Box, Button, LinkV2 } from '@island.is/island-ui/core'
+import { CompleteImage } from '../assets/CompleteImage'
+export const Submitted = (props: OJOIFieldBaseProps) => {
+  const path = window.location.origin
+  const isLocalhost = path.includes('localhost')
+  const href = isLocalhost
+    ? `http://localhost:4200/minarsidur/umsoknir#${props.application.id}`
+    : `${path}/minarsidur/umsoknir#${props.application.id}`
```

Comment:
This is a direct copy of a line in the `RedirectToServicePortalFormField.tsx`. Consider extracting to a utility function, similar `getBaseUrlForm()` in the` /portal/admin/application-system/src/shared/utils.ts`

Reply:
Its a copy because that component will redirect when mounted not on click.

Reply:
I mean the string conditional stuff